src/devices/tuya.ts

```diff
@@ -1497,6 +1517,30 @@ const definitions: Definition[] = [
             return {'l1': 1, 'l2': 1};
         },
     },
+    {
+        fingerprint: tuya.fingerprint('TS0601', ['_TZE200_nh9m9emk']),
+        model: 'TS0601-2-gang-switchboard',
+        vendor: 'TuYa',
+        description: '2 gang switch',
+        exposes: [
+            e.switch().withEndpoint('l1').setAccess('state', ea.STATE_SET),
+            e.switch().withEndpoint('l2').setAccess('state', ea.STATE_SET),
+        ],
+        fromZigbee: [fz.ignore_basic_report, legacy.fromZigbee.tuya_switch],
+        toZigbee: [legacy.toZigbee.tuya_switch_state],
```

Comment:
For all converters, don't use legacy anymore. Use only datapoints fromZigbee/toZigbee conveters, example: https://github.com/Koenkk/zigbee-herdsman-converters/blob/4a8ab6fb8c19ba30d3596321b8a3fa2e5661036c/src/devices/tuya.ts#L1434